raw_section_snooper_vocab.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+"""
+    raw_section_snooper - looks for ANY sections (XML elemnent tagged "section"?),
+        within each type:
+        - entry: ;shows template Id, concepts and names if present,
+        - entry/organizer: ;shows template Id, concepts and names if present,
+        - title
+        - code
+        - value TBD
+
+"""
+
+"""
+
+"""
+import argparse
+import re
+import xml.etree.ElementTree as ET  # https://docs.python.org/3/library/xml.etree.elementtree.html
+from xml_ns import ns
+from vocab_map_file import oid_map
+import vocab_maps
+import pandas as pd
+#experimental codeSystem lookup, likely cannot use it in Foundry
+from oid_lookup import get_oid_title
+
+
+# Global DataFrame to hold codes found
+vocab_codes = pd.DataFrame(columns=['tag', 'code', 'codeSystem', 'codeSystemName', 'displayName', 'templateId'])
+
+def get_code_name(codeSystem, codeSystemName, code, concept_df):
+    """
+    Retrieves the concept name from a concept DataFrame based on the given code system, code system name, and code.
+    Returns 'n/a' if no match is found.
+    """
+    if codeSystemName:
+        vocabulary_id = re.sub(r' CT', '', codeSystemName)  # Remove unwanted text from codeSystemName
+        concept_query = f"concept_code == '{code}' and vocabulary_id == '{vocabulary_id}'"
```

Comment:
I would more often write lines 37 and 38 as one line. If you see it like this, you can tell I separated it out to print the query and debug something.